source/catalog.html

```diff
@@ -46,19 +46,23 @@
         </picture>
       </a>
       <section class="intro intro--other">
+        <h1 class="visually-hidden">Фото и видео Седоны</h1>
         <img class="intro__text intro__text--other" src="img/sedona.svg" width="229" height="39" alt="Sedona">
       </section>
     </header>
     <main class="page-main">
       <div class="page-main__header description">
-        <h1 class="description__title description__title--catalog">Фото и видео</h1>
+        <h2 class="description__title description__title--catalog">Фото и видео</h2>
         <p class="description__text">Не можете решиться на путешествие из-за курса? Фотографии помогут вам забыть о политике и экономике.</p>
       </div>
       <section class="catalog page-main__catalog">
         <ul class="catalog__list">
           <li class="catalog__item photo-card photo-card--first">
             <a class="photo-card__link photo-card__link--first" href="#bigfoto">
               <picture>
+                <source type="image/webp" srcset="img/photo-1-desktop.webp 1x, img/photo-1-desktop@2x.webp 2x">
+                <source type="image/webp" srcset="img/photo-1-tablet.webp 1x, img/photo-1-tablet@2x.webp 2x">
+                <source type="image/webp" srcset="img/photo-1-mobile.webp 1x, img/photo-1-mobile@2x.webp 2x">
```

Comment:
Для webp также нужно указывать разрешение экрана, для которого используется данное изображение:
```suggestion
                <source type="image/webp" media="(min-width: 1200px)" srcset="img/photo-1-desktop.webp 1x, img/photo-1-desktop@2x.webp 2x">
                <source type="image/webp" media="(min-width: 768px)" srcset="img/photo-1-tablet.webp 1x, img/photo-1-tablet@2x.webp 2x">
                <source type="image/webp" srcset="img/photo-1-mobile.webp 1x, img/photo-1-mobile@2x.webp 2x">
```